wp-lazy-loading.php

```diff
@@ -122,15 +122,12 @@ function wp_lazy_loading_enabled( $tag_name, $context ) {
 /**
  * Filters specific tags in post content and modifies their markup.
  *
- * This function performs the following replacements:
- *
- * * add 'srcset' and 'sizes' attributes to 'img' tags
- * * add 'loading' attributes to 'img' tags
+ * This function adds `srcset`, `sizes`, and `loading` attributes to `img` HTML tags.
  *
  * @since (TBD)
  *
- * @see wp_image_add_srcset_and_sizes()
- * @see wp_image_add_loading()
+ * @see wp_image_tag_add_loading_attr()
+ * @see wp_image_tag_add_srcset_and_sizes_attr()
```

Comment:
I wonder whether we should name these `wp_img_tag_*`, after all that's the actual tag.

Reply:
Sounds good.